### PR TITLE
added simple and stacked lists

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added

- Simple Lists
- Stacked Lists

Here is the Preview : https://deploy-preview-22--deluxe-paletas-5b14f0.netlify.app/documentation/docs

### CODE REVIEW

**Review by rushikesh3299 — APPROVED** [review]
Looks good to me

File: documentation/docs.css [hunk 1]
```diff
@@ -245,6 +245,26 @@ iframe {
...
 }
-.txt-utils-name{
-    margin: 1rem;
-}
\ No newline at end of file
+.txt-utils-name {
+  margin: 1rem;
+}
+
+/* Lists */
```
Comment (GeetaPogul) [inline documentation/docs.css:252]: remove commented code

File: documentation/docs.css [hunk 2]
```diff
@@ -245,6 +245,26 @@ iframe {
...
+.lists-wrapper {
+  width: 12.5rem;
+  padding: 1rem;
+  margin: 2rem auto;
+  border-radius: 5px;
+  border: 1px solid var(--txt-grey-400);
+}
+.list-heading {
+  margin: 10px;
```
Comment (GeetaPogul) [inline documentation/docs.css:261]: use rem instead of px

**Review by GeetaPogul — APPROVED** [review]
your code looks good and clean
good to be merge

**Pull request merged**